source/install/ee-install.rst

```diff
@@ -52,11 +52,11 @@ GitLab Omnibus runs the open source Mattermost Team Edition. To upgrade to Matte
 
  - Go to ``/etc/gitlab/gitlab.rb`` and set the following line to false
 
-   .. code-block:: text
+.. code-block:: text
 
     mattermost['enable'] = false
 
- - Run `sudo gitlab-ctl reconfigure` to apply the updated setting
+- Run `sudo gitlab-ctl reconfigure` to apply the updated setting
```

Comment:
This bullet point is misaligned compared to the bullet point above ("Go to /etc/gitlab/gitlab.rb and set the following line to false")

Reply:
Thanks, I've revised that and it's fixed. :)

Reply:
It seems the bullet point is still misaligned here.